.gitignore

```diff
@@ -17,4 +17,5 @@
 /yarn-error.log
 
 .byebug_history
-.env
\ No newline at end of file
+.env
+.ruby-version
```

Comment:
To be honest, you shouldn't add `.ruby-version` to `.gitignore`, since it is very useful.
[Here is an article about .ruby-version file](https://blog.bigbinary.com/2018/05/07/rails-5_2-adds-ruby-version-file-and-ruby-version-to-gemfile-by-default.html)